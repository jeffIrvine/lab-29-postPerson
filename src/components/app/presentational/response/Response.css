.Response {
  height: 10em;
  border: 5px black;
  margin-right: .3vw;

}