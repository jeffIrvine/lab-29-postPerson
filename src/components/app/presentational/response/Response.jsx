import React from 'react';
import PropTypes from 'prop-types';
import styles from './Response.css';

function Response({ response }) {
  return (
    <div className={styles.Response}>
      <h2>Response</h2>
      {response}
    </div>
  );
}

Response.propTypes = {
  response: PropTypes.string.isRequired
};

export default Response;
