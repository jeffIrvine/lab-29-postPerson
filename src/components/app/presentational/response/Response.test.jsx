import React from 'react';
import Response from './Response';
import { render, cleanup } from '@testing-library/react';

describe('Response component', () => {
  afterEach(() => cleanup());
  it('renders Response', () => {
    const { asFragment } = render(<Response
      response="" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
