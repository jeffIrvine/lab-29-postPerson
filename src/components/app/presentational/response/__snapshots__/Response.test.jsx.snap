// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Response component renders Response 1`] = `
<DocumentFragment>
  <div
    class="Response"
  >
    <h2>
      Response
    </h2>
  </div>
</DocumentFragment>
`;
